src/main/resources/com/cloudbees/plugins/credentials/CredentialsStoreAction/CredentialsWrapper/sidepanel.jelly

```diff
@@ -27,10 +27,10 @@
   <l:side-panel>
     <l:tasks>
     <j:set var="url" value="${h.getNearestAncestorUrl(request,it)}"/>
-      <l:task icon="images/24x24/up.png" href="../.." title="${%backto(it.domain.displayName)}" contextMenu="false"/>
-      <l:task icon="images/24x24/setting.png" href="${url}/update" title="${%Update}" permission="${it.domain.parent.UPDATE}"/>
-      <l:task icon="images/24x24/edit-delete.png" href="${url}/delete" title="${%Delete}" permission="${it.domain.parent.DELETE}"/>
```

Comment:
These are coming from core so they should remain as is, at least until the core dep is new enough that we can use SVGs from https://github.com/jenkinsci/jenkins/pull/5663.

Reply:
Few options here I guess:

1. bump the minimum now
2. add a method to CredentialsWrapper that determines if the version is new enough and then choose dynamically
3. do this, which temporarily copies the svgs to this plugin
4. leave with non sharp pngs, not ideal

Reply:
> bump the minimum now

To at least 2.308 I guess. I am inclined to do that as it is the simplest option.

Reply:
When I created this PR, I've chosen to use approach 3, hence there are additions. I thought that would be a better temporary and lesser evil solution, until LTS includes 2.308 and you can rely on SVGs natively.

Reply:
> until LTS includes 2.308

I do not think we care. It is fine to release the plugin with a dep on a weekly. After all, users who really care about PNG vs. SVG would be unhappy with the current LTS anyway!

Reply:
True I guess, if you like blurry images, you should stay on the LTS (for now) with the PNGs.